libs/shared/ui-components/src/Form/utils.ts

```diff
@@ -12,3 +18,50 @@ export const getFormHeader = (props: TFormProps, state: FormStates): string => {
       return title;
   }
 };
+
+/**
+ * Retrieves any GCLID & UTM parameters saved to sessionStorage.
+ * GCLID capture disabled for now.
+ *
+ * @returns {ParamValues} - Encapsulated GCLID/UTM parameters
+ */
+export const getParamValues = (): ParamValues => {
```

Comment:
I know naming is hard, but "param values" is really generic, hard to search for. `getTrackingParams`?

Reply:
V good point, will change.

Reply:
Ok, I renamed everything -- see what you think